src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -68,6 +69,16 @@ private void OnParseCommand(string value)
             }
         }
 
+        /// <summary>
+        /// Read file from path and print its content in base64
+        /// </summary>
+        [ParseFunction("File path to content base64")]
+        private string? FilePathToContentBase64(string path)
+        {
+            if (!File.Exists(path)) return null;
+            return Convert.ToBase64String(File.ReadAllBytes(path));
+        }
```

Comment:
To be honest, I don't think we need this command for CLI because it's always possible to `cat file.txt | base64`.

Reply:
It's also possible but the command is a bit longer.

Reply:
Why we need it?